src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,16 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine[] array = new Machine[3];
```

Comment:
```suggestion
        Machine[] machines = new Machine[3];
```

Reply:
Ok, I will keep in mind when creating the name of the array